crates/wysiwyg/src/tests/test_formatting.rs

```diff
@@ -29,6 +29,7 @@ fn selecting_and_bolding_multiple_times() {
         &model.state.dom.to_string(),
         "<strong>aa</strong>bb<strong>cc</strong>"
     );
+    assert_eq!(&model.state.dom.to_raw_text(), "aabbcc");
```

Comment:
This would be better as a separate test. Would also be worth checking e.g. an empty Dom, some nested tags, some empty tags etc.